src/NodaTime.Test/DateIntervalTest.cs

```diff
@@ -76,10 +76,10 @@ public void Equals_DifferentCalendars()
         {
             LocalDate start1 = new LocalDate(2000, 1, 1);
             LocalDate end1 = new LocalDate(2001, 6, 19);
-            // This is a really, really similar calendar to ISO - the dates could differ by week of year,
-            // but that's all.
-            LocalDate start2 = start1.WithCalendar(CalendarSystem.GetGregorianCalendar(1));
-            LocalDate end2 = end1.WithCalendar(CalendarSystem.GetGregorianCalendar(1));
+            // This is a really, really similar calendar to ISO, but we do distinguish.
+            // TODO: Should we?
```

Comment:
Okay. Let's discuss and make that change separately though. (I think it might be a bit confusing for users aware of the Gregorian calendar system, but not aware of ISO-8601.)